frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/simulated-region-feature-manager.ts

```diff
@@ -142,19 +142,16 @@ export class SimulatedRegionFeatureManager
             return false;
         }
         if (
-            ['vehicle', 'personnel', 'material', 'patient'].includes(
-                droppedElement.type
-            )
+            droppedElement.type === 'vehicle' ||
+            droppedElement.type === 'personnel' ||
+            droppedElement.type === 'material' ||
+            droppedElement.type === 'patient'
         ) {
             this.exerciseService.proposeAction(
                 {
                     type: '[SimulatedRegion] Add Element',
                     simulatedRegionId: droppedOnSimulatedRegion.id,
-                    elementToBeAddedType: droppedElement.type as
-                        | 'material'
-                        | 'patient'
-                        | 'personnel'
-                        | 'vehicle',
+                    elementToBeAddedType: droppedElement.type,
```

Comment:
[I already brought this up previously.](https://github.com/hpi-sam/digital-fuesim-manv/pull/626/files#r1093237887)

If someone would have mistyped one of the keys in the array that was previously present, you wouldn't have noticed this. Therefore I changed it back to the old condition.
The type checks now work correctly as `droppedElement` is typed since your refactoring.